tagging-status/testfiles/lineno/lineno-01.tex

```diff
@@ -0,0 +1,16 @@
+\DocumentMetadata
+  {
+    lang=en-US,
+    pdfversion=2.0,
+    pdfstandard=ua-2,
+    testphase={phase-III,math,title,table,firstaid}
+  }
+\documentclass{article}
+\usepackage{lineno}
+
+\begin{document}
+\begin{linenumbers}
+bla bla
+\end{linenumbers}
```

Comment:
one could think of covering a few other typical usecases, e.g linenos with lists, in boxes ... maybe as separate test files

Reply:
@FrankMittelbach I've added more tests for lineno, but I'm confused about the "in boxes" since the lineno doc says
> Lines in any kind of boxes cannot be numbered

Reply:
I might remember incorrectly but  while it doesn't automaticaly does it it has a feature to enable numbering locally. It does: \internallinenumbers

Reply:
I see. Test for that is added for that